ob1k-core/src/main/java/com/outbrain/ob1k/client/dispatch/StaticDoubleDispatchStrategy.java

```diff
@@ -0,0 +1,27 @@
+package com.outbrain.ob1k.client.dispatch;
+
+import com.outbrain.ob1k.concurrent.Try;
+
+/**
+ * A simple double dispatch strategy with static threshold
+ *
+ * @author lifey
+ */
+public class StaticDoubleDispatchStrategy extends AbstractDoubleDispatchStrategy {
+
+  private final long durationMs;
+
+  public StaticDoubleDispatchStrategy(final long durationMs) {
+    this.durationMs = durationMs;
+  }
+
+  @Override
+  public long provideInterval() {
+    return durationMs;
+  }
+
+  @Override
+  public void dispatchResult(final Try result, final long startTimestamp) {
```

Comment:
@lifey agree. change to 'onDispatchResult', I think it's more clear now.
@haski both calls results are being sent to this method. It doesn't return anything as it's an event method. The reason it's exists is because its mostly for future implementation, such as DynamicDoubleDispatch, that will need result status (thus the Try), and the roundtrip time